### ISSUE TITLE
Remove the incorrect video from signup

### ISSUE THREAD

**Issue by DevDeckardCain** [dtube/dtube#306]
The promotional video that is shown after a user creates their account should be removed. It's full of misinformation.

**Comment by 9mido** [comment]
Can you explain what the misinformation is?

**Comment by skzap** [comment]
Yes this video is old. I'm working on simplifying the signup process completely, the current one has proven to be really bad.

**Comment by DevDeckardCain** [comment]
> Can you explain what the misinformation is?

https://ipfs.io/ipfs/QmRZow9T4nxsZ6nk12ARWSvNaKkhx9EoRDyenFS7bBwAvt?filename=WhyBecomeADTuber.mp4

That is the original video. Its filled with misinformation and inaccuracies.

**Comment by DevDeckardCain** [comment]
> Yes this video is old. I'm working on simplifying the signup process completely, the current one has proven to be really bad.

Right on. Thank you.
